### ISSUE TITLE
Sort Hotkey not working anymore

### ISSUE THREAD

**Issue by SeoliteQ** [Salandora/SophisticatedStorage#17]
**Describe the bug**
Clean install on fabric 1.20.1 with only modernkeybinding, sophisticated storage, backpack and core installed. 

**To Reproduce**
1. Open storage container
2. Press Hotkey for sorting storage
3. Nothing happens (yes, I moved around items in the storage chest and added them at random places)

**Expected behavior**
Storage chest should be sorted.

**Versions**
Sophisticated Storage 0.8.60+mc1.20.1-build.104

**Comment by Salandora** [comment]
Does it work when you click the gui button?
If yes check if something conflicts with the hotkey.
If no I more and more get the suspicion that one of the recent bugfixes is a bug monster...

**Comment by SeoliteQ** [comment]
Yes, the GUI button works just fine. I do prefer to use the hotkey though. There should not be any conflict as I'm testing this on a clean version with only fabric, modern keybinding and your fabric ports.

Yes... The storage controller stopped working for me once again, too, on my heavily modded version. I haven't found the conflicting mod (out of the 300sth mods installed) yet though.

**Comment by Salandora** [comment]
Did you rebind the sort key by any chance and if so what did you rebind it to?
Also when you press the hotkey for sorting is your mouse over a slot of the storage?
I just double checked the code and it does check if you hover over a storage related slot.
It won't sort if your mouse is outside the gui or on a player inventory slot.

**Comment by SeoliteQ** [comment]
I rebound it to "R", which apparently does not work on a clean install even though there's no visible conflict in the keybind menu. When I set it to middle mouse button, it works again.

**Comment by Salandora** [comment]
Can confirm this bug happens when rebound to a keyboard key due to an oversight when redoing the mkb compatibility.

Btw in the newest version mkb is not required anymore.

EDIT: Bug will still happen without mkb. Just notices this could be wrongly interpreted.

**Comment by SeoliteQ** [comment]
I need modernkeybinding for another mod either way. Without it, none of the keybinds for your fabric ports work for me anymore (probably due to conflicting configs with other mods). I am slowly getting used to using the mouse middle button for sorting anyway now :) 

On a separate note, I figured out the controller issue. It appears that sophisticated chests become broken when copied from one world to another via mcaedit. I had to set up my storage system with newly generated chests, which was not fun, but now the storage controller works like a charm again.
